Checks/PWR023/example.c

```diff
@@ -1,4 +1,4 @@
-// PWR023: add ‘restrict’ for pointer function parameters to hint the compiler
+// PWR023: add ‘restrict’ for pointer function arguments to hint the compiler
```

Comment:
nit:

```suggestion
// PWR023: Add ‘restrict’ for pointer function arguments to hint the compiler
```

This belongs in a different commit, though!

Reply:
Done!